packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -107,4 +107,9 @@ export interface IChoiceGroupOption {
    */
   labelId?: string;
 
+  /**
+   * Optional identifier for automating the clickable control in the choice field.
+   */
+  automationId?: string;
```

Comment:
Or better, can we extend the interface from HTMLAttribute<something> and mix in the extra things? That way we can support passing in data-automation-id.